zerver/webhooks/github/view.py

```diff
@@ -165,7 +165,7 @@ def get_issue_body(helper: Helper) -> str:
         action=action,
         url=issue["html_url"].tame(check_string),
         number=issue["number"].tame(check_int),
-        message=issue["body"].tame(check_none_or(check_string)),
+        message=issue["body"].tame(check_none_or(check_string)) if action == "opened" else None,
```

Comment:
What are the possible values of `action` here?

If there are values other than "opened" and "assigned", then this will drop the issue body from those too. That might be fine, but we should choose explicitly.